samples/msgext-ai-sentiment-analysis/nodejs/views/sentimentModule.ejs

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Run build pipeline on this branch.

Reply:
Executed Build Pipeline : **https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5899869239**